fracas/rules.cpp

```diff
@@ -254,6 +254,7 @@ static GameDefs g = {
 	200,	// MIN_DESTROY_POINTS,
 	34,	// MAX_DESTROY_PERCENT
 	0, // HALF_RIDING_BONUS
+	GameDefs::REPORT_FORMAT_TEXT,	// REPORT_FORMAT
```

Comment:
This make sure noone gets the JSON format until we are ready.  I do set it in the unittest/rules.cpp so I when we eventually do end-to-end/integrations tests there as well as unit tests it will just work.